.travis.yml

```diff
@@ -1,13 +1,19 @@
 language: rust
-sudo: false
```

Comment:
Does this need to be removed? I added it to get the "container" infrastructure for Travis-CI.